unity-environment/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System.Collections;
```

Comment:
Please ensure that all code lines are within 100 characters.

Reply:
My IDE default to 120. Are we going with 100 for everything?

Reply:
80 for python (PEP 8 standard) / C++ and 100 for C#/Java.